### DISCUSSION TITLE
Удаление файлов в циклах

### DISCUSSION THREAD

**Discussion by Re-Requeste** [totumonline/totum-ru-issues-and-discussions#533] [Как это сделать?] [answered]
Добрый день, не могу понять как удалять файлы.

В таблице расчетная циклов, в хедере есть поле "Загрузить". Файлы идут в папку как и описано [тут](https://ru.docs.totum.online/fields#type-file) , при ручном загрузке и ручном удаление все ок, но мне бы хотелось чтобы файлы сами удалялись через определенное время через cron.

Суть проблемы, что не могу подобрать команду для удаления файлов с HDD. 
clear просто очищает, delete удаляет строку в строчной части. 

У временных таблиц, есть префикс !tmp, но к сожалению я не могу загружать через временные таблицы

В какую сторону изучать подскажите пожалуйста

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Хм. Когда вы кроном ставите пустое значение в поле загрузить разве файл связанный с этим полем не удаляется с диска?

**Reply by erpic-ru** [reply]
Ну, кстати, да и так все удаляется с диска. Отмена

**Reply by Re-Requeste** [reply]
Да, проверил = : set(table: 'test'; cycle:2 ;  field: 'h_download' = $#lc ; log: true)
Удалил а так же удалил файл с диска, я пробовал через clear кнопку не срабатывал :) 
Спасибо

**Reply by ttm-alexeypolunin** [reply]
Clear очищает установленные "ручные" когда в поле есть Код, но поле дост на редактирование и было изменено. Надо изменить названия этих функций на pin и unpin.
